docs/index.Rmd

```diff
@@ -1680,7 +1704,7 @@ Aggregator is a function or sequence or map of functions which accept dataset as
 
 Where map is given as an input or result, keys are treated as column names.
 
-Grouped dataset is ungrouped after aggreation. This can be turned off by setting `:ungroup` to false. In case you want to pass additional ungrouping parameters add them to the options.
+Grouped dataset is ungrouped after aggreation. This can be turned off by setting `:ungroup?` to false. In case you want to pass additional ungrouping parameters add them to the options.
```

Comment:
I've found one more typo. `aggreation` -> `aggregation`.

Reply:
Might as well bundle that into this PR, so I made the change.

Reply:
(I didn't read this text carefully!  I tried to experiment with ungrouping with `aggregate`, and it didn't work with `:ungroup`; then I noticed `:ungroup?` in the code example.)

Reply:
Probably there are more such unfortunate things in the text... :/

Reply:
Yeah, can't be helped.

(Do you need me to squash the commits at this point?)